src/Service/Extractor/Factory/ExtractorFactory.php

```diff
@@ -19,11 +22,46 @@ public function __invoke(ContainerInterface $container): ExtractorInterface
         /** @var array $config */
         $config = $container->get(ModuleConfig::class);
 
-        $extractor = $container->get(
-            $config['extractor'] ?? AnnotationExtractor::class
-        );
-        assert($extractor instanceof ExtractorInterface);
+        $extractors = [];
+
+        $extractorFromConfig = $this->extractorFromConfig($container, $config);
```

Comment:
I understand that the functionality of the extractor added via configuration was not added in this PR, but I will ask a question here:

Why did you provide the possibility to add only one extractor via configuration? In my opinion, this is a completely unmotivated restriction.

Your code is fully ready to work with any number of extractors.

Reply:
There is no real reason for that. There was just no need for it at this time but I will add it.

Reply:
released version 5.5.0 with extractor configuration can now be an array of extractors